spec/unit/source/git_spec.rb

```diff
@@ -157,9 +187,9 @@
       expect(bn.dirname).to eq 'mybranch'
     end
 
-    it "prepends the source name when prefixing is on" do
-      bn = described_class.new('mybranch', {:prefix => true, :sourcename => 'foo'})
-      expect(bn.dirname).to eq 'foo_mybranch'
+    it "prepends the prefix name when prefixing is overridden" do
```

Comment:
The flow here changed a bit.   `true` won't actually make it into the `BranchName` class anymore, due to the `derive_prefix` function in source/base.pp

https://github.com/brettswift/r10k/blob/feature/master/multi_tenant_prefix/lib/r10k/source/base.rb#L36-L44

So in BranchName, the only possible values are `false` or `:String`. 

I'm updating the documentation for `initialize` in this class to reflect that.